tests/gha-run-k8s-common.sh

```diff
@@ -325,22 +325,20 @@ function deploy_rke2() {
 
 function deploy_microk8s() {
 	sudo snap install microk8s --classic
+	sudo usermod -a -G microk8s ${USER}
+	mkdir -p ~/.kube
+	sudo microk8s kubectl config view --raw > ~/.kube/config
+	sudo chown ${USER}:${USER} ~/.kube/config
```

Comment:
Sometimes I got an error on this where a directory permission is not properly configured. Why don't we run `sudo chown -R ${USER}:${USER} ~/.kube` instead? Thanks.

Reply:
Just opinion. You can ignore it. 😉